### ISSUE TITLE
Error when trying to push tags to the remote (for iP)

### ISSUE THREAD

**Issue by wilburrito** [nus-cs2103-AY2122S1/forum#75]
Currently trying to tag my commit and push it to the remote using the CLI on IntelliJ. I am able to tag it locally, but when I try to push the tag to the remote I get an error:
 "error: src refspec Level-8 matches more than one"

I've already checked that this tag does not exist on the remote, and even tried to delete it (to which it returned an error "unable to delete 'Level-8': remote ref does not exist".

Has anyone encountered this before? Any advice on fixing this? 

Thanks in advance!

**Comment by ErnestCuong** [comment]
I am not very sure but I think a tag cannot have the same name as another branch. I could not push my commit in a branch to remote because I have a tag of the same name in remote already, after deleting the tag I was able to push. Did you by any chance name one of your branch as Level-8?

**Comment by marcuspeh** [comment]
Was it tagged locally already (just not pushed to remote)? Perhaps u can try to delete the tag locally and tagging again.

**Comment by wilburrito** [comment]
@ErnestCuong @marcuspeh Thanks for the reply guys.

I've already tried deleting the tag locally and re-tagging it, and then trying to push it to the remote again, only to get back the same error.

I think I may have misnamed one of my branches yesterday (I called it Level-8 instead of branch-Level-8). I changed it on the remote but I did not change it locally. 

I have now renamed my wrongly-named branch on my local repo as well, and I am able to successfully push my tags to the remote repo. Cheers! Appreciate the quick reply.
